### ISSUE TITLE
Configured storage account in binder does only create the container, not the blob

### ISSUE THREAD

**Issue by pgroene** [Azure/azure-webjobs-sdk-script#1195]
Blob is not created in binder when StorageAccountAttribute is used in Binder. Only the container is created.

#### Repro steps

Provide the steps required to reproduce the problem

Create a function:
using Microsoft.Azure.WebJobs;
using Microsoft.Azure.WebJobs.Host.Bindings.Runtime;

```
public static async  void Run(string  myBlob, string name, Binder binder, TraceWriter log )
{
    log.Info($"C# Blob trigger function processed: {name} {myBlob} -> peer test3");
    string path = "testcontainer/xxx";
    log.Info($"path: {path}");

    var attributes = new Attribute[]
    {    
        new BlobAttribute(path),
        new StorageAccountAttribute("storageaccount")
    };

    using (var writer = await binder.BindAsync<TextWriter>(attributes))
    {
        writer.Write(myBlob);
    }
}
```

with bindings:
```
{
  "bindings": [
    {
      "name": "myBlob",
      "type": "blobTrigger",
      "direction": "in",
      "path": "test/{name}",
      "connection": "storageaccount"
    }
  ],
  "disabled": false
}
```

confiruge the storage account: storageaccount in the appsettings

put a blob in the test container

the container testcontainer is created, the blob is not created

#### Expected behavior

Blob is created

#### Actual behavior

Only the container, without blob is created

#### Known workarounds

Do this without  the attribute  new StorageAccountAttribute("storageaccount")

then the container and blob are created in the function its storage account

#### Related information 

functions version 1

**Comment by mathewc** [comment]
Do you see any errors in your function logs? Likely the blob write is failing for some reason. You're saying that the blob container is correctly created in the secondary storage account (meaning the account info is valid)?

This is known to work and we have tests exercising it, so we just have to figure out what is happening in your scenario :) https://stackoverflow.com/questions/39855409/how-do-i-use-binder-to-perform-dynamic-bindings-in-my-c-sharp-function

**Comment by pgroene** [comment]
Any suggestions on how to get more logging. I used that question/answer to create the function idd. I have done the following steps:

Reconfigured the connectionstrings => no blob
Changed the name of the blob to larger and smaller path => no result
Created a new Storage account without any logging, encryption etc (standard mp) => no blob
created a new function app with the mp and added the function to see if there was something in the arm templates => no blob
Changed the output storageaccount to the same as the function storage account => no blob

in all scenarios the container was created, no error in the log files. I'm 100% sure the connection strings are correct. I'm testing in West Europe.
